### ISSUE TITLE
be_polyelectrolyte model docs (Trac #553)

### ISSUE THREAD

**Issue by RichardHeenan** [SasView/sasview#685]
Model contains a "contrast factor" that is not explained in the docs, and may have the wrong units.

Migrated from http://trac.sasview.org/ticket/553
```json
{
    "status": "closed",
    "changetime": "2016-10-08T01:08:28",
    "_ts": "2016-10-08 01:08:28.777681+00:00",
    "description": "Model contains a \"contrast factor\" that is not explained in the docs, and may have the wrong units.",
    "reporter": "richardh",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Bug Fixing",
    "time": "2016-03-22T19:38:18",
    "component": "SasView",
    "summary": "be_polyelectrolyte model docs",
    "priority": "critical",
    "keywords": "",
    "milestone": "SasView 4.1.0",
    "owner": "richardh",
    "type": "defect"
}
```

**Comment by butlerpd** [comment]
Trac update at `2016/09/05 02:49:51`: 
* **butler** commented:

as part of the last model documentation sweep this should be checked.  Moving back to 4.0 for now


* **butler** changed milestone from "SasView Next Release +1" to "SasView 4.0.0"

**Comment by butlerpd** [comment]
Trac update at `2016/09/05 14:41:04`: **butler** changed priority from "major" to "critical"

**Comment by butlerpd** [comment]
Trac update at `2016/09/13 13:31:00`: **butler** changed milestone from "SasView 4.0.0" to "SasView 4.1.0"

**Comment by butlerpd** [comment]
Trac update at `2016/10/08 01:08:28`: 
* **butler** commented:

In changeset 5df888c6540361f7d5cf02845ce9d6a422974042:

```text
#!CommitTicketReference repository="sasmodels" revision="5df888c6540361f7d5cf02845ce9d6a422974042"
added defintions, fixed units and gnerally cleaned up
be_polyelectrolyte.py model documentation. closes #685
```


* **butler** changed resolution from "" to "fixed"
* **butler** changed status from "new" to "closed"
